java-frontend/src/main/java/org/sonar/java/se/xproc/MethodYield.java

```diff
@@ -56,7 +56,7 @@
   final ExplodedGraph.Node node;
   private final Map<String, Map<String, Set<Flow>>> cachedFlows = new HashMap<>();
   private final MethodBehavior behavior;
-  List<ConstraintsByDomain> parametersConstraints;
+  public List<ConstraintsByDomain> parametersConstraints;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make parametersConstraints a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)